pyvista/core/pointset.py

```diff
@@ -919,15 +919,26 @@ def n_lines(self) -> int:
     def n_verts(self) -> int:
         """Return the number of vertices.
 
+        A vertex is a single element cell referencing a single point.
```

Comment:
```suggestion
        A vertex is a 0D cell, which is usually a cell that references one point,
        a vtkVertex.  It can also be a vtkPolyVertex.
```